.travis.yml

```diff
@@ -37,6 +37,12 @@ matrix:
     - php: 7.0
       env: PHPCS=1 DEFAULT=0
 
+    - php: 7.0
```

Comment:
Do we need both, or should we just have 7.1 here (and also in allowed failure list then). The result should be the same pretty much either way, right?